sql/core/src/main/scala/org/apache/spark/sql/DataFrameNaFunctions.scala

```diff
@@ -395,22 +395,24 @@ final class DataFrameNaFunctions private[sql](df: DataFrame) {
 
   private def fillMap(values: Seq[(String, Any)]): DataFrame = {
     // Error handling
+    var resolved: Map[String, Any] = Map()
     values.foreach { case (colName, replaceValue) =>
       // Check column name exists
-      df.resolve(colName)
+      val resolvedColumn = df.resolve(colName)
 
       // Check data type
       replaceValue match {
         case _: jl.Double | _: jl.Float | _: jl.Integer | _: jl.Long | _: jl.Boolean | _: String =>
-          // This is good
+        // This is good
```

Comment:
Nit: this indent was correct

Reply:
Thanks for catching, I shall fix it and also add comments for using additional map.

Reply:
Taken care of.